kibana_logger/__init__.py

```diff
@@ -18,6 +21,7 @@ class KibanaLogger(object):
     CEE = "@cee: "
 
     def __init__(self, preset=None):
+        self.log_level = LogLevel[os.getenv('LOG_LEVEL', default='INFO')]
```

Comment:
as it's a library `LOG_LEVEL` will very likely conflict with environment variable used by the application integrating you.

Reply:
for library it's usually preferable to set things through setter/factories rather than env variables.